torch/_C/_distributed_c10d.pyi

```diff
@@ -25,6 +25,7 @@ class GradBucket:
     def get_per_parameter_tensors(self) -> List[Tensor]: ...
     def is_the_last_bucket_to_allreduce(self) -> bool: ...
     def set_tensor(self, tensor: Tensor) -> None: ...
+    def get_model_params_for_bucket(self) -> List[Tensor]: ...
```

Comment:
NI: I think we can just call this `get_params`, since all the data in this `GradBucket` class is for the bucket only. No need to add extra `for_bucket`.

Reply:
I'll call it `get_model_params` for now, otherwise could be confusing with existing `get_per_parameter_tensors`, which is actually gradients, which will be renamed in a follow up diff